summer/compute.py

```diff
@@ -1,24 +1,41 @@
+"""
+Optimized 'hot' functions used by CompartmentalModel and its runners.
+"""
 import numba
 import numpy as np
 
 
```

Comment:
Split accumulate_flow_contributions into two functions because numba gets sad when either flow map array is empty.